layouts/default.vue

```diff
@@ -0,0 +1,18 @@
+<script setup>
+import { ElContainer, ElHeader, ElMain, ElFooter } from "element-plus";
+</script>
+<template>
+  <div class="common-layout">
+    <el-container class="container">
+      <el-header></el-header>
+      <el-main><slot /></el-main>
+      <el-footer></el-footer>
+    </el-container>
+  </div>
+</template>
+<style>
+.container {
```

Comment:
you could create your own component `base-container` using windi css 

https://windicss.org/utilities/layout/container.html#container-1